boards/arm/swan_r5/swan_r5.dts

```diff
@@ -38,7 +38,7 @@
 
 
 	zephyr,user {
-		no-pull-gpios = <&gpioe 6 GPIO_PUSH_PULL>;
+		dischrg-gpios = <&gpioe 6 GPIO_ACTIVE_LOW>;
```

Comment:
> (I assume it does by pulling the DISCHRG line low?

yes, but it should be open drain to prevent back-feeding U1 (1V8 regulator). Note that U1 input is specified as 2.5V-3.3V and when it is at 2.5V, configuring push-pull and driving this line high will result in 800uA from the pin, which is not insignificant.

Reply:
Good point, will fix